### PR TITLE
build: wip

### PR DESCRIPTION
**Pull request by matthew-garrett**
build: updates mobile nav styling and icons

### CODE REVIEW

File: packages/shared/wallet/src/network-selector/network-selector.tsx [hunk 1]
```diff
@@ -138,16 +156,181 @@ export function PortfolioNetworkSelector({
       sx={sx}
       isPortfolio
       onNetworkChange={(network) => portfolioStore.setNetwork(network)}
+      hideOnMobile
     />
   );
 }
 
+export function MobileNetworkSelector() {
```
Comment (jeffywu) [inline packages/shared/wallet/src/network-selector/network-selector.tsx]: Should we split this into a different file?

File: packages/shared/wallet/src/network-selector/network-selector.tsx [hunk 2]
```diff
@@ -138,16 +156,181 @@ export function PortfolioNetworkSelector({
...
+            boxShadow: theme.shape.shadowLarge(),
+            border: theme.shape.borderStandard,
+            background: theme.palette.background.default,
+          },
+        }}
+      >
+        <NetworkInnerWrapper
+          hideNetWorth={hideNetWorth}
+          theme={theme}
+          sx={{ marginTop: '72px', padding: '16px !important' }}
```
Comment (jeffywu) [inline packages/shared/wallet/src/network-selector/network-selector.tsx]: Can we use theme.spacing throughout here?

File: packages/shared/web/src/header/header.tsx [hunk 3]
```diff
@@ -60,6 +63,19 @@ export function Header({ children }: HeaderProps) {
     setInLocalStorage('hideSubGraphError', true);
   };
 
+  useEffect(() => {
```
Comment (jeffywu) [inline packages/shared/web/src/header/header.tsx:66]: Should this be done in the App.tsx file instead?

Comment (matthew-garrett) [reply packages/shared/web/src/header/header.tsx:66]: Imo we keep this in the header. We want the local state update happening in the header component. It guarantees that the mobile nav is applied immediately.

File: packages/shared/web/src/header/header.tsx [hunk 4]
```diff
@@ -60,6 +63,19 @@ export function Header({ children }: HeaderProps) {
...
+      setIsMobileView(false);
+    }
+    const handleResize = () => {
+      setIsMobile(window.innerWidth <= 768);
+    };
+    window.addEventListener('resize', handleResize);
+    return () => window.removeEventListener('resize', handleResize);
+  }, [setIsMobileView, isMobileView]);
+
   window.addEventListener('scroll', () => {
```
Comment (jeffywu) [inline packages/shared/web/src/header/header.tsx]: Should this also go into a useEffect and have a removeEventListener as well?

**Pull request merged**